NERDweb/templates/ips.html

```diff
@@ -300,6 +300,7 @@ <h1>Search IP addresses by ...</h1>
     <span>{{ip.events_meta.total if ip.events_meta else 0}}</span><span>{{ip._evt_info.n_nodes}}</span><span>{{ip._evt_info.n_cats}}</span>
     {% if ip._showable_misp_events %}<br><span class="misp">+ {{ ip._showable_misp_events }} MISP events</span>{% endif %}
     {% if ip.otx_pulses %}<br><span class="otx">+ {{ ip.otx_pulses|length }} OTX pulses</span>{% endif %}
+    {% if ip.dshield %}<br><span class="dsheild">+ {{ ip.dshield|length }} DShield events</span>{% endif %}
```

Comment:
typo: `dsheild` -> `dshield`

Reply:
You can't use `length`. With the way DShield data are stored, you just get the number of days with some alerts. We want the total number of alerts in the last 90 days (as with Warden).
You must go through the array and sum up the number of alerts for all the dates. Any you probably need to do this in the backend (`nerd_main.py`, e.g. around line 1074) to avoid complex computations in the template.